include/robot_interfaces/robot_log_entry.hpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * \file
+ * \copyright Copyright (c) 2020, Max Planck Gesellschaft.
+ */
+#pragma once
+
+#include <time_series/interface.hpp>
+
+#include <robot_interfaces/status.hpp>
+
+namespace robot_interfaces
+{
+/**
+ * @brief Robot log entry used for binary log format.
+ *
+ * Contains all the robot data of one time step.
+ */
+template <typename Action, typename Observation>
+struct RobotLogEntry
```

Comment:
same for here if this is also specific for binary datatype

Reply:
This is currently only used for the binary log but the struct itself is not specific to that, it could also be used for other formats (e.g. with cereal it would be easy to use JSON or XML instead of the binary format).  Therefore, I would not add the "Binary" here.

Reply:
Okay, yes, that makes sense